### PR TITLE
Fatih/more explicit tests

### PR DESCRIPTION
**Pull request by aktasfatih**
PT ID: #177734507

I have changed the tests to show the checks for the camelCase keys.

### CODE REVIEW

**Review by horgh — CHANGES_REQUESTED** [review]
Looks good! Some minor comments.

File: src/webServiceClient.spec.ts [hunk 1]
```diff
@@ -1,5 +1,7 @@
+import { Insights } from '@maxmind/geoip2-node';
```
[the remark this answers is not in the document]
Comment (aktasfatih) [reply src/webServiceClient.spec.ts:1]: I have removed that too.

File: src/webServiceClient.spec.ts [hunk 2]
```diff
@@ -34,17 +36,252 @@ describe('WebServiceClient', () => {
...
+      expect(got!.subscores!.emailTenure).toEqual(0.11);
+      expect(got!.subscores!.ipTenure).toEqual(0.12);
+      expect(got!.subscores!.issuerIdNumber).toEqual(0.13);
+      expect(got!.subscores!.orderAmount).toEqual(0.14);
+      expect(got!.subscores!.phoneNumber).toEqual(0.15);
+      expect(got!.subscores!.shippingAddress).toEqual(0.99);
+      expect(got!.subscores!.shippingAddressDistanceToIpLocation).toEqual(0.16);
+      expect(got!.subscores!.timeOfDay).toEqual(0.17);
+
+      // Note: Couldn't write a check for the default item in the subscores array
```
[the remark this answers is not in the document]
Comment (aktasfatih) [reply src/webServiceClient.spec.ts:264]: It was showing up in the debugger. But, it doesn't anymore😅. I have removed that too.

File: src/webServiceClient.spec.ts [hunk 3]
```diff
@@ -74,8 +503,8 @@ describe('WebServiceClient', () => {
       ${'credit_card'}
       ${'email'}
       ${'shipping_address'}
-    `('handles missing response $property', ({ property }) => {
-      expect.assertions(1);
+    `('handles missing response $property', async ({ property }) => {
```
[the remark this answers is not in the document]
Comment (aktasfatih) [reply src/webServiceClient.spec.ts:506]: You are right. I have added the tests for those cases.

**Review by horgh — APPROVED** [review]

**Pull request merged**